core/templates/pages/exploration-editor-page/modal-templates/state-version-history-modal.component.ts

```diff
@@ -148,18 +156,23 @@ export class StateVersionHistoryModalComponent
       return;
     }
     const diffData = this.versionHistoryService.getBackwardStateDiffData();
-    if (diffData.oldVersionNumber !== null) {
+    if (
```

Comment:
I think this needs a bit of explanation for why diffData.oldVersionNumber !== null is not sufficient.

Reply:
(Did you miss this?)

Reply:
Changed this block now. PTAL. Thanks!